HopsanGUI/MainWindow.cpp

```diff
@@ -804,6 +804,14 @@ void MainWindow::createActions()
     mHelpPopupTextMap.insert(mpExportToSimulinkAction, "Export model to Simulink S-function.");
     connect(mpExportToSimulinkAction, SIGNAL(hovered()), this, SLOT(showToolBarHelpPopup()));
 
+    mpExportToExe_32Action = new QAction(QIcon(QString(ICONPATH) + "Hopsan-ExportExe.png"), tr("Export to Executable Model (32-bit)"), this);
```

Comment:
I would prefer if we start using the icons in .svg format

Reply:
I agree, but I think we should then change all icons in a separate PR.

Reply:
Yes